Xamarin.Forms.Platform.MacOS/Renderers/BoxViewRenderer.cs

```diff
@@ -32,8 +39,19 @@ protected override void SetBackgroundColor(Color color)
 		{
 			if (Element == null || Control == null)
 				return;
+
 			Control.WantsLayer = true;
 			Control.Layer.BackgroundColor = color.ToCGColor();
 		}
+
+		void SetCornerRadius(CornerRadius cornerRadius)
+		{
+			if (Element == null)
+				return;
+
```

Comment:
here's a good pattern for setting that property in all renderers:

```
if (cornerRadius == default(CornerRadius)
    RestoreToDefault(...); //you can not just return, as the property might have been set to a value earlier
else
    ApplyCornerRadius(...);
```
might be overkill in this case, as BoxView has no native counterpart, and we know the the default is 0;